### PR TITLE
Fix of keys update alert

### PR DESCRIPTION
**Pull request by Amuhar**

### CODE REVIEW

File: prometheus/alertmanager.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+global:
```
Comment (F4ever) [inline prometheus/alertmanager.yml:1]: Alertmanager configs added, but no description and guide in README how to test alerts

Comment (Amuhar) [reply prometheus/alertmanager.yml:1]: added in readme

File: README.md [hunk 2]
```diff
@@ -32,12 +32,21 @@ For running locally in container run
...
 
+## Metrics and alerts
+
+To launch Prometheus, Grafana, and Alertmanager, execute the following command:
+
+```
+docker-compose -f docker-compose.metrics.yml up
+```
+
+For configuring Grafana, navigate to http://localhost:8000/dashboards. Here, you can import dashboards from the ./grafana folder.
```
Comment (F4ever) [inline README.md:46]: To configure Grafana dashboards, navigate to `URL` and import dashboards from the `./grafana` folder.
Export them after you done?

Comment (Amuhar) [reply README.md:46]: i think import is enough, we dont update outr grafana based on this dashboards
we do visa versa

Comment (Amuhar) [reply README.md:46]: and use this dashboards for local development

File: alerts/keys-api-outdated-keys.non-empty-db.rule.yml [hunk 3]
```diff
@@ -0,0 +1,13 @@
+groups:
+  - name: Keys API. Update on non empty db
+    rules:
+      - alert: KeysApiOutdatedKeysNonEmptyDB
+        expr: |
+          (time() - lido_keys_api_last_update_timestamp{}) >= 10*60 and lido_keys_api_last_update_timestamp{} > 0
```
Comment (F4ever) [inline alerts/keys-api-outdated-keys.non-empty-db.rule.yml:6]: Maybe add `time() - process_start_time_seconds{}) < 15 * 60` here?

**Review by F4ever — APPROVED** [review]

File: alerts/keys-api-outdated-keys.non-empty-db.rule.yml (hunk 3 above)
Comment (Amuhar) [reply alerts/keys-api-outdated-keys.non-empty-db.rule.yml:6]: we discussed , that here 15 it is enough

**Review by infloop — APPROVED** [review]

**Pull request merged**